src/Moryx.Maintenance.Web/Pages/Maintenance.cshtml

```diff
@@ -18,4 +18,10 @@
   <script src="/_content/Moryx.Maintenance.Web/bundle.js" type="module"></script>
 }
 
-<div id="app"></div>
\ No newline at end of file
+<div id="app" style="height: 100%;
```

Comment:
Could you place the styling in the maintenance.scss file as well? as the body element is styled in there as well, this should work, shoudn't it?